fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1572,56 +1551,96 @@ public void replayAddPartition(PartitionPersistInfo info) throws DdlException {
     public void dropPartition(Database db, Table table, DropPartitionClause clause) throws DdlException {
         CatalogUtils.checkTableExist(db, table.getName());
         OlapTable olapTable = (OlapTable) table;
+        PartitionInfo partitionInfo = olapTable.getPartitionInfo();
```

Comment:
These two pieces of codes should be placed below `Locker locker = new Locker();`

Reply:
sorry, should be below `Preconditions.checkArgument(locker.isWriteLockHeldByCurrentThread(db));`, After ensuring that the current thread holds the lock, we can operate on the meta-object.